nikola/utils.py

```diff
@@ -2028,3 +2028,111 @@ def read_from_config(self, site, basename, posts_per_classification_per_language
         args = {'translation_manager': self, 'site': site,
                 'posts_per_classification_per_language': posts_per_classification_per_language}
         signal('{}_translations_config'.format(basename.lower())).send(args)
+
+
+# Moved to global variable to avoid recompilation
+# of regex every time re_meta() is called.
```

Comment:
Regex fun fact: there’s a cache, for 512 entries if memory serves. It probably doesn’t get recompiled. (Don’t change this back)